qiita_pet/templates/study_description.html

```diff
@@ -580,4 +601,21 @@
   {% end %}
 </div>
 
+
+<div class="modal fade delete-study" tabindex="-1" role="dialog" id="delete-study">
+  <div class="modal-dialog modal-sm">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h3>Deleting:<br/></h3><h4>{{study_title}}</h4>
+        You will only be able to delete a study that has no data associated with it.
+      </div>
+      <div class="modal-body">
+        To continue you need to write the alias name of the study:
+          <input type="text" name="study-alias" id="study-alias">
+          <a class="btn btn-danger glyphicon glyphicon-trash" onclick="delete_study()"></a>
```

Comment:
A comment that @josenavas made is that we are not being careful about what the HTML elements can and can't do, and this is one of those cases. If there's no text in the text box, then the button should not be enabled.